docs/guides/connectingclusters.md

```diff
@@ -186,7 +208,7 @@ spec:
 
 ### Receiving Events
 
-With all components being setup CloudEvents should be flowing from `PingSource` at the sender cluster to the `event-display` service at the receiver cluster. We can make sure by looking at the receiving service logs.
+With all components being setup CloudEvents should be flowing from `WebhookSource` at the sender cluster to the `event-display` service at the receiver cluster. We can make sure by looking at the receiving service logs.
```

Comment:
If we are going to remove the use of PingSource in favor of WebHookSource, we need to expand the documentation here to include steps &| sample requests to the WebHookSource. As of right now the documentation is written as if things should just be working with no farther steps.

Reply:
You're right that I forgot to add examples for the webhook requests to this guide. adding!